required/examples/basics/container-pivot.js

```diff
@@ -1,41 +1,32 @@
-var renderer = PIXI.autoDetectRenderer(800, 600,{backgroundColor : 0x1099bb});
-document.body.appendChild(renderer.view);
-
-// create the root of the scene graph
-var stage = new PIXI.Container();
+var app = new PIXI.Application(800, 600, {backgroundColor : 0x1099bb});
+document.body.appendChild(app.view);
 
 var container = new PIXI.Container();
 
-stage.addChild(container);
+app.stage.addChild(container);
 
-for (var j = 0; j < 5; j++) {
+// Create a new texture
+var texture = PIXI.Texture.fromImage('required/assets/basics/bunny.png');
 
-    for (var i = 0; i < 5; i++) {
-        var bunny = PIXI.Sprite.fromImage('required/assets/basics/bunny.png');
-        bunny.x = 40 * i;
-        bunny.y = 40 * j;
-        container.addChild(bunny);
-    };
-};
+// Create a 5x5 grid of bunnies
+for (var i = 0; i < 25; i++) {
+    var bunny = new PIXI.Sprite(texture);
+    bunny.anchor.set(0.5);
+    bunny.x = (i % 5) * 40;
+    bunny.y = Math.floor(i / 5) * 40;
+    container.addChild(bunny);
+}
 
 // move container to the (200, 150) 
-container.position.x = 200;
-container.position.y = 150;
-// (93, 98.5) is center of center bunny sprite in local container coordinates
-// we want it to be in (200, 150) of global coords
-container.pivot.x = 80 + 26 * 0.5;
-container.pivot.y = 80 + 37 * 0.5;
-
-// start animating
-animate();
+container.x = 400;
+container.y = 300;
 
-function animate() {
+// Center bunny sprite in local container coordinates
+container.pivot.x = container.width * 0.5;
+container.pivot.y = container.height * 0.5;
 
-    requestAnimationFrame(animate);
-  
+// Listen for animate update
+app.ticker.add(function() {
     //rotate the container!
     container.rotation -= 0.01;
```

Comment:
Would we want to show best practice by using time delta here for a frame rate independent transform?